### ISSUE TITLE
run without debugging command does not work

### ISSUE THREAD

**Issue by alberto-ingenito** [microsoft/vscode#202504]
Type: <b>Bug</b>

run-> run without debugging: nothing happens at all. Not even terminal opens.

VS Code version: Code 1.85.1 (0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2, 2023-12-13T09:49:37.021Z)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz (8 x 1805)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.39GB (2.11GB free)|
|Process Argv|--crash-reporter-id a15ec5c6-e344-4863-9af7-d1670eb4bc09|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (10)</summary>

Extension|Author (truncated)|Version
---|---|---
FreeMarker|dco|0.0.9
csharp|ms-|2.15.30
vscode-dotnet-runtime|ms-|2.0.0
python|ms-|2023.22.1
vscode-pylance|ms-|2023.12.1
cmake-tools|ms-|1.16.32
cpptools|ms-|1.18.5
cpptools-extension-pack|ms-|1.3.0
cmake|twx|0.0.17
json|Zai|2.0.2

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593:30376534
py29gd2263:30899288
vsclangdf:30486550
c4g48928:30535728
azure-dev_surveyone:30548225
a9j8j154:30646983
0bi6i642:30933247
pythongtdpath:30769146
i26e3531:30792625
welcomedialogc:30910334
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
h48ei257:30885898
pythontbext0:30879054
accentitlementst:30887150
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
dsvsc019acf:30933242

```

</details>

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's missing some basic information or in some other way doesn't follow our [issue reporting guidelines](https://aka.ms/vscodeissuereporting). Please take the time to review these and update the issue.

Happy Coding!

**Comment by roblourens** [comment]
Please give specific details. What language? What launch config? Your code should run but depending on what it is, you might not see a result, what are you expecting?

**Comment by alberto-ingenito** [comment]
Language: Python. I did not define any launch configuration.I just wrote
python code and 'run without debugging' as usual. It worked with same code
and same command other times but suddenly stop to work this time. With
'stop to work' I mean that terminal is not opened and neither is launched
the usual command to run the script: &
'C:\Users\me\AppData\Local\Microsoft\WindowsApps\python3.11.exe'
'c:\Users\me\.vscode\extensions\ms-python.python-2023.22.1\pythonFiles\lib\python\debugpy\adapter/../..\debugpy\launcher'
'55006' '--' 'path\to\script.py'

The command started to work again after several VSCode restarts
